webapp/channels/src/components/admin_console/system_user_detail/team_list/team_list_dropdown.tsx

```diff
@@ -19,66 +18,59 @@ type Props = {
-        const showMakeTeamAdmin = !isAdmin && !isGuest;
-        const showMakeTeamMember = !isMember && !isGuest;
-
-        return (
-            <MenuWrapper
-                isDisabled={this.props.isDisabled}
+    return (
+        <MenuWrapper isDisabled={isDisabled}>
+            <button
+                type='button'
+                id={`teamListDropdown_${team.id}`}
+                className='dropdown-toggle theme color--link style--none'
+                aria-expanded='true'
             >
-                <button
-                    type='button'
-                    id={`teamListDropdown_${team.id}`}
-                    className='dropdown-toggle theme color--link style--none'
-                    aria-expanded='true'
+                <span className='SystemUserDetail__actions-menu-icon'><EllipsisHorizontalIcon/></span>
+            </button>
+            <div>
+                <Menu
+                    openLeft={true}
+                    openUp={false}
+                    ariaLabel={intl.formatMessage({id: 'team_members_dropdown.menuAriaLabel', defaultMessage: 'Change the role of a team member'})}
                 >
-                    <span className='SystemUserDetail__actions-menu-icon'><EllipsisHorizontalIcon/></span>
-                </button>
-                <div>
-                    <Menu
-                        openLeft={true}
-                        openUp={false}
-                        ariaLabel={localizeMessage('team_members_dropdown.menuAriaLabel', 'Change the role of a team member')}
-                    >
-                        <Menu.ItemAction
-                            id='makeTeamAdmin'
-                            show={showMakeTeamAdmin}
-                            onClick={() => this.props.doMakeUserTeamAdmin(team.id)}
-                            text={localizeMessage('team_members_dropdown.makeAdmin', 'Make Team Admin')}
-                        />
-                        <Menu.ItemAction
-                            show={showMakeTeamMember}
-                            onClick={() => this.props.doMakeUserTeamMember(team.id)}
-                            text={localizeMessage('team_members_dropdown.makeMember', 'Make Team Member')}
-                        />
-                        <Menu.ItemAction
-                            id='removeFromTeam'
-                            show={true}
-                            onClick={() => this.props.doRemoveUserFromTeam(team.id)}
-                            text={localizeMessage('team_members_dropdown.leave_team', 'Remove from Team')}
-                            buttonClass='SystemUserDetail__action-remove-team'
-                        />
-                    </Menu>
-                </div>
-            </MenuWrapper>
-        );
-    }
-}
+                    <Menu.ItemAction
+                        id='makeTeamAdmin'
+                        show={showMakeTeamAdmin}
+                        onClick={() => doMakeUserTeamAdmin(team.id)}
```

Comment:
Let's move the three `onClick`s to useCallback, to avoid passing a new function on every render.